tools/buildtools/protein_structure_rebuilders/pulchra/pulchra.xml

```diff
@@ -0,0 +1,85 @@
+<tool id="pulchra" name="PULCHRA recontruction of all atom proteins" version="@TOOL_VERSION@+galaxy@GALAXY_VERSION@">
+    <description>from reduced representations</description>
+    <macros>
+        <token name="@TOOL_VERSION@">3.06</token>
+        <token name="@GALAXY_VERSION@">0</token>
+    </macros>
+    <requirements>
+        <requirement type="package" version="3.06">pulchra</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+
+    ln -s '$structure_input' ./structure_input.${structure_input.ext} &&
+	pulchra 
+	
+	##inputs
+	./structure_input.${structure_input.ext}
+     #if $skip_sidechain:
+    -s
+    #end if
+
+    -v
+ 
+    ]]></command>
+    <inputs>
+        <param name="structure_input" type="data" format="pdb" label="Structure file" help="In PDB format. This input is usually an 'alpha trace' or other incomplete structure file."/>
+        <param name="skip_sidechain" type="boolean" label="Skip sidechain reconstruction?" help="By default, both sidechains and backbone will be reconstructed otherwise."/>
```

Comment:
```suggestion
        <param argument="-s" name="skip_sidechain" type="boolean" truevalue="-s" falsevalue="" label="Skip sidechain reconstruction?" help="By default, both sidechains and backbone will be reconstructed otherwise."/>
```